plugins/scripts/upload/trns_transform_mzML_LCMS_to_MetaboliteAtlas2_MAFileInfo.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+# standard library imports
+import os
+import sys
+import logging
+import datetime
+
+# 3rd party imports
+import simplejson
+
+# KBase imports
+try:
+    import biokbase.Transform.script_utils as script_utils
+except ImportError:
+    from . import script_utils
+
+# Metatlas Imports
+from metatlas import mzml_loader
```

Comment:
This dependency is not found in KBase environment. You should add something like this into https://github.com/kbase/transform/blob/develop/deps/pylib.sh :
-----------------------------------------------------------------------------------------
pip install git+https://github.com/metabolite-atlas/metatlas
But this command doesn't work unfortunately with such error:
Downloading/unpacking git+https://github.com/metabolite-atlas/metatlas
  Cloning https://github.com/metabolite-atlas/metatlas to /tmp/pip-PAM0Ze-build
  Running setup.py (path:/tmp/pip-PAM0Ze-build/setup.py) egg_info for package from git+https://github.com/metabolite-atlas/metatlas
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ImportError: No module named pymzml
    cp: missing destination file operand after `obo.patch'
    Try `cp --help' for more information.
    install.sh: line 9: pushd: no other directory
    install.sh: line 10: obo.patch: No such file or directory
    rm: cannot remove `obo.patch': No such file or directory
    install.sh: line 12: popd: directory stack empty
Downloading/unpacking pymzml (from metatlas==0.2)
  Downloading pymzml-0.7.5.zip (115kB): 115kB downloaded
  Running setup.py (path:/tmp/pip_build_root/pymzml/setup.py) egg_info for package pymzml
Requirement already satisfied (use --upgrade to upgrade): simplejson in <...> (from metatlas==0.2)
Installing collected packages: pymzml, metatlas
  Running setup.py install for pymzml
  Running setup.py install for metatlas
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ImportError: No module named pymzml
    cp: missing destination file operand after `obo.patch'
    Try `cp --help' for more information.
    install.sh: line 9: pushd: no other directory
    install.sh: line 10: obo.patch: No such file or directory
    rm: cannot remove `obo.patch': No such file or directory
    install.sh: line 12: popd: directory stack empty